server/endpoints/api/admin/index.js

```diff
@@ -133,7 +133,13 @@ function apiAdminEndpoints(app) {
 
       const newUserParams = reqBody(request);
       const { user: newUser, error } = await User.create(newUserParams);
-      response.status(200).json({ user: newUser, error });
+      
+      if (!newUser) {
+        // e.g., password is too short
+        response.status(400).json({ user: newUser, error });
+      } else {
+        response.status(200).json({ user: newUser, error });
+      }
```

Comment:
can we just do a small change to refactor to a one liner since we dont reuse the var?
```javascript
response.status(newUser ? 200 : 400).json({ user: newUser, error });
```